docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,471 @@
+# Basic Usage
+
+Dojo is a reactive architecture concerned with constantly modifying and rendering the current state of an application. In simple systems this can happen at a local level and widgets can maintain their own state. However, as a system becomes more complex the need to better compartmentalize and encapsulate data and create a clean separation of concerns quickly grows.
+
+Stores provide a clean interface for storing, modifying, and retrieving data from a global object using a unidirectional data flow. Common patterns such as asynchronous retrieval, middleware, and undo patterns are built in. This allows widgets to remain focused on their primary roles of providing a visual representation of themselves and listening for user interactions.
+
+## The Store
+
+The store hold the global, atomic state for the entire application. It is created when the application is created and defined in the `Registry` with an injector.
+
+> main.ts
+
+```ts
```

Comment:
Do you think that we should use the helper from `StoreInjector`, `registerStoreInjector` for the example:

```ts
import { registerStoreInjector } from '@dojo/framework/store/StoreInjector';
import Store from '@dojo/framework/stores/Store';
import { State } from './interfaces';

const store = new Store<State>();
const registry = registerStoreInjector(store);
```

Reply:
Totally. I forgot about that :)